visualization/open3d_vis_utils.py

```diff
@@ -96,7 +98,7 @@ def create_all_frustums_open3d(wTi_list: List[Pose3], calibrations: List[Cal3Bun
         py = K[1, 2]
         img_w = px * 2
         img_h = py * 2
-        frustum_obj = ViewFrustum(fx, img_w, img_h)
+        frustum_obj = ViewFrustum(fx, img_w, img_h, frustum_ray_len=frustrum_ray_len)
```

Comment:
thanks for the PR, @travisdriver. It looks good, except for a small typo on `frustum` here and throughout these two files

Reply:
fixed